CHANGELOG.adoc

```diff
@@ -17,6 +17,7 @@
 - https://github.com/eclipse-syson/syson/issues/298[#298] [metamodel] Fix metamodel to manage root namespaces.
 - https://github.com/eclipse-syson/syson/issues/329[#329] [services] Ignore root namespace with no name during qualified name resolution 
 - https://github.com/eclipse-syson/syson/issues/337[#337] [diagrams] Fix direct edit of single digit cardinalities.
+- https://github.com/eclipse-syson/syson/issues/348[#348] [diagrams] Succession edge is not well formed
```

Comment:
Could you please rephrase this message and be more explicit please? The Succession edge is not really the problem here, it is the semantic elements created by the tool allowing to create a Succession edge, right?

Reply:
Right.
What do you think about: 
`The semantic representation of the Succession edge is not correct.`

Reply:
ok